github/pkg/adapter/adapter.go

```diff
@@ -34,6 +34,8 @@ import (
 const (
 	GHHeaderEvent    = "GitHub-Event"
 	GHHeaderDelivery = "GitHub-Delivery"
+	GHCEHeaderEvent    = "event"
```

Comment:
You could take a look at https://github.com/cloudevents/spec/blob/master/adapters/github.md and see if you agree with @duglin